src/main/java/org/recordrobotics/charger/control/DoubleControl.java

```diff
@@ -24,6 +27,25 @@ public double getDriveLat() {
 		return -_gamepad2.getLeftX();
 	}
 
+	@Override
+	public ClawState getClawTurn() {
+		// Out mimics button-like behavior
+		boolean cube = _gamepad2.getLeftTriggerAxis() > TRIGGER_THRESHOLD;
+		boolean cone = _gamepad2.getRightTriggerAxis() > TRIGGER_THRESHOLD;
+
+		if (_gamepad2.getLeftTriggerAxis() > TRIGGER_THRESHOLD)
+			_lTrig = !_lTrig;
+
+		if (_gamepad2.getRightTriggerAxis() > TRIGGER_THRESHOLD)
+			_rTrig = !_rTrig;
+
+		// Cube takes precedence
+		if (cube || cone){
```

Comment:
The OR seems out of place, would it not make sense to check for the individually right away?